### PR TITLE
SAK-42790 Improve assignment progress bar style

### PR DESCRIPTION
**Pull request by josecebe**

### CODE REVIEW

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakai/sakai/pullRequest?prid=4532664&bid=15256337)

File: library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss [hunk 1]
```diff
@@ -547,9 +547,23 @@
...
+
+		.progress-cols {
+			position: absolute;
+			width: 100%;
+			background: no-repeat;
+		}
+
+		.progress-bar-empty {
+			background-color: transparent;
+			color: #212121;
```
Comment (ottenhoff) [inline library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss]: This is same as $text-color ? can you use variable instead?

**Comment by ern** [comment]
Nice visual improvement

File: library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss (hunk 1 above)
Comment (josecebe) [reply library/src/morpheus-master/sass/modules/tool/assignments/_assignments.scss]: Done!

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakai/sakai/pullRequest?prid=4532664&bid=15256337)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakai/sakai/pullRequest?prid=4532664&bid=15256337)

**Comment by steveswinsburg** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/sakai/sakai/pullRequest?prid=4532664&bid=15256337)

**Pull request merged**